packages/database/src/repository/DAG.ts

```diff
@@ -91,7 +90,70 @@ async function initialize(props: DAGInitProps): Promise<void> {
   })
 }
 
+/**
+ * Toggle a Module's status between placed and hidden.
+ * @param {string} moduleCode
+ */
+async function toggleModule(thisDag: DAG, moduleCode: string): Promise<DAG> {
```

Comment:
this is my bad, we were deciding between mutating `this` and returning a new object, but here we have both, so don't need to `const res` and return I believe